easybuild/scripts/rpath_args.py

```diff
@@ -36,6 +36,26 @@
 import sys
 
 
+def is_new_existing_path(new_path, paths):
+    """
+    Check whether specified path exsits and is a new path compared to provided list of paths.
+    """
+
+    # assume path is new, until proven otherwise
+    res = True
+
+    if os.path.exists(new_path):
+        for path in paths:
+            if os.path.exists(path) and os.path.samefile(new_path, path):
+                res = False
+                break
+    else:
+        # path doesn't exist
+        res = False
+
+    return res
+
+
```

Comment:
I can't help thinking this could be shorter.
```
    if not os.path.exists(new_path):
        return False

    for path in paths:
        if os.path.exists(path) and os.path.samefile(new_path, path):
            return False

    return True
```
though it's a different coding style, so i'll leave this as a suggestion

Reply:
We usually don't do inline `return`s, but have just a single `return` per function